core/src/main/java/org/apache/calcite/rel/rel2sql/SqlImplementor.java

```diff
@@ -1337,6 +1345,16 @@ public Clause maxClause() {
       return maxClause;
     }
 
+    /** return whether {@code Clause.GROUP_BY} is contained in {@code clauses} */
```

Comment:
This can be replaced by clauses.contains(Clause.GROUP_BY)

Reply:
Thanks, I will change it.